packages/aws-cdk-lib/aws-events/test/event-bus.test.ts

```diff
@@ -40,11 +40,13 @@ describe('event bus', () => {
     // WHEN
     new EventBus(stack, 'Bus', {
       eventBusName: 'myEventBus',
+      description: 'myEventBus',
```

Comment:
Could you separate it and create a new test because the name of this test is `'named event bus'`?